internal/envoy/v3/listener.go

```diff
@@ -202,18 +204,44 @@ func (b *httpConnectionManagerBuilder) ConnectionShutdownGracePeriod(timeout tim
 }
 
 func (b *httpConnectionManagerBuilder) DefaultFilters() *httpConnectionManagerBuilder {
+
 	b.filters = append(b.filters,
 		&http.HttpFilter{
 			Name: wellknown.Gzip,
+			ConfigType: &http.HttpFilter_TypedConfig{
+				TypedConfig: protobuf.MustMarshalAny(&envoy_compressor_v3.Compressor{
+					CompressorLibrary: &envoy_core_v3.TypedExtensionConfig{
+						Name: "gzip",
+						TypedConfig: &any.Any{
+							TypeUrl: "type.googleapis.com/envoy.extensions.compression.gzip.compressor.v3.Gzip",
```

Comment:
is there anywhere we can source the type URLs as constants from?

Reply:
Hmm not sure if go-control-plane has some, I could extract into constants in this file though.